content/en/docs/tasks/administer-cluster/encrypt-data.md

```diff
@@ -259,14 +259,10 @@ The following table describes each available provider.
     Data is encrypted by data encryption keys (DEKs) using AES-GCM; DEKs
     are encrypted by key encryption keys (KEKs) according to configuration
     in Key Management Service (KMS).
-    Kubernetes defaults to generating a new DEK at API server startup, which is then
```

Comment:
Removed these references because these docs are specific to v1.29+ and `KMSv2KDF` flag is locked to `true`.